core/src/main/java/com/linecorp/armeria/common/RequestContextWrapper.java

```diff
@@ -183,6 +183,27 @@ public MeterRegistry meterRegistry() {
         return delegate().meterRegistry();
     }
 
+    @Override
+    public void cancel(Throwable cause) {
+        delegate().cancel(cause);
+    }
+
+    @Override
+    public void cancel() {
+        delegate().cancel();
+    }
+
+    @Override
+    public void timeoutNow() {
+        delegate().timeoutNow();
+    }
+
+    @Override
+    @Nullable
+    public Throwable cancellationCause() {
+        return delegate().cancellationCause();
+    }
+
```

Comment:
Would you pull up `isCancelled()` and `isTimedOut()` too?

Reply:
Don't think we need to since they're implemented as default methods.

Reply:
Ah... OK 😅